src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/RetryPolicyUtils.cs

```diff
@@ -26,9 +30,25 @@ public static class RetryPolicyUtils
         /// Blocklist of non-transient errors that should stop retry during data transfer operations
         /// </summary>
         private static readonly HashSet<int> _nonRetryableDataTransferErrors;
+        /// <summary>
+        /// Max intervals between retries to wake up serverless instances.
+        /// </summary>
+        private const int _serverlessMaxIntervalTime = 30;
```

Comment:
Add the unit of measure (in the name and/or the XML comment), e.g. `_serverlessMaxIntervalTimeInSeconds`.